week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,20 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
 console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// This program will join together two or more strings.
```

Comment:
Are you certain it can join more than two? How?

Reply:
I meant concatenates. It does not join.